Talent.Services.Profile/Controllers/ProfileController.cs

```diff
@@ -138,63 +142,321 @@ public async Task<IActionResult> IsUserAuthenticated()
         public async Task<IActionResult> GetLanguages()
         {
             //Your code here;
-            throw new NotImplementedException();
+            //throw new NotImplementedException();
+            var userId = _userAppContext.CurrentUserId;
```

Comment:
Can you verify first if  **_userAppContext.CurrentUserId;** has value or not before doing anything